GitUI/CommandsDialogs/RsaUtil.cs

```diff
@@ -0,0 +1,365 @@
+﻿using System;
+using System.IO;
+using System.Numerics;
+using System.Security.Cryptography;
+using System.Text;
+using System.Windows.Forms;
+
+using JetBrains.Annotations;
+
+namespace GitUI.CommandsDialogs
+{
+	/// <summary>
+	/// RSA key formats manip.
+	/// </summary>
+	public static class RsaUtil
```

Comment:
I also could not come up yet.  I am more iinteresting in testing System.Cryptography on mono.